ccloud/start-docker.sh

```diff
@@ -4,42 +4,35 @@
 . ../utils/helper.sh
 
 echo ====== Verifying prerequisites
-check_jq || exit
-
-# File with Confluent Cloud configuration parameters: example template
-#   $ cat ~/.ccloud/config
-#   bootstrap.servers=<BROKER ENDPOINT>
-#   ssl.endpoint.identification.algorithm=https
-#   security.protocol=SASL_SSL
-#   sasl.mechanism=PLAIN
-#   sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
-#   # Confluent Cloud Schema Registry
-#   basic.auth.credentials.source=USER_INFO
-#   schema.registry.basic.auth.user.info=<SR API KEY>:<SR API SECRET>
-#   schema.registry.url=https://<SR ENDPOINT>
-#   # Confluent Cloud KSQL
-#   ksql.endpoint=https://<KSQL ENDPOINT>
-#   ksql.basic.auth.user.info=<KSQL API KEY>:<KSQL API SECRET>
-export CONFIG_FILE=~/.ccloud/config
-
+check_ccloud_version 1.0.0 \
+  && print_pass "ccloud version ok" \
+  || exit 1
```

Comment:
I don't believe these exit calls will ever be invoked given the design of the check_ccloud_version which will exit on version check failure prior to getting to this "or".  I think it's fine to make it like the following until the functions change to only return codes instead of exit the shell.

```suggestion
check_ccloud_version 1.0.0 \
  && print_pass "ccloud version ok"
```

Reply:
For consistency I will leave it as-is (because of how it's used in other scripts).